### PR TITLE
[Discussion] Incorrect lambda binding generation

### PR DESCRIPTION
**Pull request by jslee02**
This PR adds a minimal test that reproduces the issue of lambda binding (see #148).

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,8 @@ add_chimera_binding(
 
 set_target_properties(test_binding PROPERTIES PREFIX "")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```
Comment (prasbg) [inline test/CMakeLists.txt]: Just out of curiosity, why is this flag necessary if the target compile options are set below?  Is there some other target that is not getting the options?

Comment (psigen) [inline test/CMakeLists.txt]: Just out of curiosity, why is this flag necessary if the target compile options are set below? Is there some other target that is not getting the options?

Comment (jslee02) [inline test/CMakeLists.txt]: I haven't dig further, but it's not compiled without it complaining `std::function` is not defined even with `<functional>` included.

**Comment by jslee02** [comment]
Closing as this is resolved by https://github.com/personalrobotics/chimera/pull/295

**Pull request closed without merging**